File: lines/lines.py
import time
import datetime

count = 0
with open('../lines.log', 'w', buffering=1) as f:
    while True:
        count += 1
        f.write(f'time: {datetime.datetime.now():%H:%M:%S %d-%m-%Y}, count: {count}\n')
        time.sleep(1)

File: lines/lines.cpp
#include <iostream>
#include <fstream>
#include <thread>
#include <chrono>
using namespace std;

int main () {
  ofstream myfile;
  myfile.open("../lines.log");
  for (int i = 0; i < 100 ; i++) {
    time_t now = time(0);
    myfile << now << ", count: " << i << "\n" << flush;
    this_thread::sleep_for(chrono::milliseconds(1000));
  }
  myfile.close();
  return 0;
}
